content/millennium/bulk-data.md

```diff
@@ -0,0 +1,95 @@
+
+* The size of the population for which data was requested
+* The number of resource types requested
+* The time range of activity being requested
+* Inherent latency due to network issues and effects from concurrent requests to the system
+
+#### Example use cases
+
+* Creating an internal clinical data warehouse for study purposes
+* Machine learning applications obtaining training data from EHR
+* Integration of population health system with an EHR system
+* Data transfer to payer database for care quality assessment
+* Submission to reportable disease (or other) registries
+
+<!-- TOC -->
+* TOC
+{:toc}
+
+## Implemented Exports
+
+Cerner supports only the Group Export operation.
+
+* [Group Export](/millennium/bulk-data/group-export/)
+
+## Registering Client Applications
+
+In order to be granted access to the Bulk Data Access API, a Millennium Bulk SMART application must be registered through Cerner's [code Console](http://fhir.cerner.com/smart/#code-console). See [Authorization](http://fhir.cerner.com/authorization/) for additional details on registering client applications and authorizing with the server.
+
+In the case of Group Export, client application developers will need to work with healthcare organizations to identify the available Groups to be exported. Organizations can manage Groups within Ignite Management Tooling.
+
+_Notes_
+
+* Authentication is required, open endpoints do not support bulk export workflows.
+* Cerner supports v1.0.1 of the [FHIR Bulk Data Access (FLATFHIR) Implementation Guide](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/)
+* Bulk data runs against an organization's database and utilizes the same services running other applications, so it is important to consider performance. Bulk data exports large amounts of data for large groups of patients, which takes more time to complete the larger the data set or patient population.
+
+## Client Errors
+
+The following are the possible client errors that can be received on API calls:
+
+1. Failing to send a required search parameter or sending a parameter with an invalid value will result in a `400 Bad Request` response.
+
+        HTTP/1.1 400 Bad Request
+
+2. Making a request without valid credentials will result in a `401 Unauthorized` response.
+
+        HTTP/1.1 401 Unauthorized
+
+3. Requesting data from an unknown instance or an instance where the application is not authorized will result in a `403 Forbidden` response.
+
+        HTTP/1.1 403 Forbidden
+
+4. Requesting a resource which does not exist will result in a `404 Not Found` response.
+
+        HTTP/1.1 404 Not Found
+
+5. Requesting a media type other than JSON will result in a `406 Not Acceptable` response.
+
+        HTTP/1.1 406 Not Acceptable
+
+        Content-Length: 0
+
+6. Performing an export kick off using unsupported search parameter values
+
+        HTTP/1.1 422 Unprocessable Entity
+
+### Lenient Workflow
```

Comment:
do we want to make it clear we support Lenient Workflow from V2

Reply:
Have added a note here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/c936d35a3ba46f521cf0e5f6addfec2b7276c599#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R94